display_list/display_list_canvas_unittests.cc

```diff
@@ -2952,7 +2952,7 @@ TEST_F(DisplayListCanvas, DrawAtlasNearest) {
       SK_ColorMAGENTA,
   };
   const sk_sp<SkImage> image = CanvasCompareTester::kTestImage;
-  const DlImageSampling sampling = DlImageSampling::kNearestNeighbor;
+  DlImageSampling sampling = DlImageSampling::kNearestNeighbor;
```

Comment:
These 3 const modifiers should probably be left in because they are locking a local variable against modification. Only "const" in argument lists is unnecessary.